### PR TITLE
v2.5 Mobile changelog

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

**Review by cpoile — APPROVED** [review]

**Review by larkox — APPROVED** [review]

**Review by enahum — APPROVED** [review]

File: source/deploy/mobile-app-changelog.md [hunk 1]
```diff
@@ -2,12 +2,45 @@
...
+ - Improved reaction time for the **Copy Text** option.
+ - Improved behavior around "Maximum call stack size exceeded" errors.
+ - Android: removed unneeded space after the list in the **Find Channels** screen.
+ - Removed unneeded group calls that were appearing in the logs.
+
+### Bug Fixes
+ - Fixed an issue where **Search** and **Recent Mentions** did not highlight saved posts.
+ - Fixed the interactive dialog radio buttons style when using the light theme.
+ - Fixed a small issue when marking threads as read.
+ - Fixed an issue where the save button sometimes did not show on the edit post screen.
```
Comment (cwarnermm) [inline source/deploy/mobile-app-changelog.md:34]:
```suggestion
 - Fixed an issue where the **Save** button sometimes did not show on the Edit Post screen.
```

File: source/deploy/mobile-app-changelog.md [hunk 2]
```diff
@@ -2,12 +2,45 @@
...
+ - Fixed theming issues in the login screen.
+ - Fixed an issue where replies sometimes seemed to be attributed to the wrong person or the wrong thread.
+ - Ensured users mentioned in message attachments are loaded by the app.
+ - Fixed issues with the user status sync.
```
Comment (cwarnermm) [inline source/deploy/mobile-app-changelog.md:38]:
```suggestion
 - Fixed issues with user status synchronization.
```

**Review by cwarnermm — APPROVED** [review]
Provided feedback inline and approving for expediency.

**Review by hmhealey — APPROVED** [review]

File: source/deploy/mobile-app-changelog.md [hunk 3]
```diff
@@ -2,12 +2,45 @@
...
+ - iPhone 5s devices and later with iOS 12.1+ is required.
+
+### Improvements
+ - Calls: Added call quality degradation logic and a banner to alert users when they are on unstable connections.
+ - Improved client error handling.
+ - Post editing now respects the ``message_source`` field.
+ - Improved reaction time for the **Copy Text** option.
+ - Improved behavior around "Maximum call stack size exceeded" errors.
+ - Android: removed unneeded space after the list in the **Find Channels** screen.
+ - Removed unneeded group calls that were appearing in the logs.
```
[the remark this answers is not in the document]
Comment (amyblais) [reply source/deploy/mobile-app-changelog.md]: Next release is fine.

**Pull request merged**